tiingo/api.py

```diff
@@ -85,59 +86,55 @@ def __init__(self, *args, **kwargs):
         }
 
         self._frequency_pattern = re.compile('^[0-9]+(min|hour)$', re.IGNORECASE)
+        self._ticker_list = None
 
     def __repr__(self):
         return '<TiingoClient(url="{}")>'.format(self._base_url)
 
-    def _is_eod_frequency(self,frequency):
+    @staticmethod
+    def _is_eod_frequency(frequency):
         return frequency.lower() in ['daily', 'weekly', 'monthly', 'annually']
 
-    # TICKER PRICE ENDPOINTS
-    # https://api.tiingo.com/docs/tiingo/daily
-    def list_tickers(self, assetTypes=[]):
-        """Return a list of dicts of metadata tickers for all supported tickers
-            of the specified asset type, as well as metadata about each ticker.
-            This includes supported date range, the exchange the ticker is traded
-            on, and the currency the stock is traded on.
-           Tickers for unrelated products are omitted.
-           https://apimedia.tiingo.com/docs/tiingo/daily/supported_tickers.zip
-           """
+    @staticmethod
+    def _get_ticker_list():
+        """Downloads the list of supported tickers."""
         listing_file_url = "https://apimedia.tiingo.com/docs/tiingo/daily/supported_tickers.zip"
         response = requests.get(listing_file_url)
         zipdata = get_zipfile_from_response(response)
         raw_csv = get_buffer_from_zipfile(zipdata, 'supported_tickers.csv')
         reader = csv.DictReader(raw_csv)
+        return [row for row in reader]
 
-        if not len(assetTypes):
-            return [row for row in reader]
-
-        assetTypesSet = set(assetTypes)
-        return [row for row in reader
-                if row.get('assetType') in assetTypesSet]
+    @staticmethod
+    def _format_response(response, fmt, object_name=None):
+        """Converts the api response into json, csv, or objects.
```

Comment:
Not sure I understand this comment.  The `object_name` cannot be a list.  Or, do you mean to clarify that the returned result can be either a single namedtuple or list thereof?

Reply:
Ah the positioning of my comment was unfortunate, I was indeed referring to the type of `objects` in the docstring rather than the object_name parameter. Your interpretation of `mean to clarify that the returned result can be either a single namedtuple or list thereof` is right :).